### DISCUSSION TITLE
solving for Bitbucket re[p access: fatal: bad boolean config value '0 for 'GIT_TERMINAL_PROMPT'

### DISCUSSION THREAD

**Discussion by RibeyeOnTheBone** [semaphoreui/semaphore#2015] [Q&A]
I am attempting my first Semaphoreui project. On task run I am blocked by this issue.

`9:40:58 AM Cloning Repository https://myname@bitbucket.org/myname/repname.git`
`9:40:58 AM Cloning into 'repository_2_5'...`
`9:40:59 AM fatal: bad boolean config value '0`
`9:40:59 AM ' for 'GIT_TERMINAL_PROMPT'`

My steps have been
* Create ssh-key using ssh-keygen 
* Put private key into KeyStore
* Put public key into Bitbucket repository Access key

If I use an invalid repo name it traps that error, so this "bad value" error seems to happening after the repo is found.

Have not used Bitbucket repository Access keys before, so I am not 100% confident of the steps I am taking.

**Comment by fiftin** [comment]
Hi,
did you see this: https://docs.semui.co/administration-guide/troubleshooting#fatal-bad-numeric-config-value-0-for-git_terminal_prompt-invalid-unit ?

**Comment by RibeyeOnTheBone** [comment]
Yes, thanks fitin, I did see that. Could not make it work, unfortunately.

Set up a "login_password" ket using my bitbucker login email as the name and the bitbucket API token as the pw.

On "repository", under "URL or path" I select https type

Error is 

fatal: unable to access 'https://xxx.xx.x:ATATT3.....looks like my access key@myrepository@bitbucket.org/myanme/myrepositoyt/': URL rejected: Port number was not a decimal number between 0 and 65535

which looks like it is using the access key where the post should be.

Make sense?

**Reply by tboerger** [reply]
The error doesn't make sense, it looks like there are mixed ssh and https clone URLs.

**Reply by fiftin** [reply]
Hi @RibeyeOnTheBone Did you solve the issue? I just experimented with Bitbucket Repo Token in wrote this doc: https://docs.semui.co/user-guide/repositories/bitbucket_access_token

**Comment by RibeyeOnTheBone** [comment]
Ah, thanks team.

I have been waylaid on another issue, but I am sure the info and instructions will fix it. I hope to complete this week.

Many thanks again
